qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -207,12 +226,15 @@ def close_tab(self, tab):
         """
         last_close = config.get('tabs', 'last-close')
```

Comment:
The problem with this is that you can also have last-close set to ignore.  In which case we don't want to run this check and prompt the user if there is only one tab but we do if there are multiple tabs.  So basically after that I got back to the current set of ifs.

Let me know if you can see some other clearer was to remove the duplication.  But this was the best I could think of.